package.json

```diff
@@ -61,4 +62,4 @@
   },
   "name": "core",
   "version": "2.3.0"
-}
\ No newline at end of file
+}
```

Comment:
Make sure, that the pre-commit hook uses eslint to check the changed codebase (see `.lintstagedrc`file)